### ISSUE TITLE
Basic Web page

### ISSUE THREAD

**Issue by VikingPingvin** [VikingPingvin/go-locker#8]
For browsing namespace/project/Jobs/artifacts

**Comment by Gerifield** [comment]
I'd go with a very simple json api (over http) for now. It's very good for automation and you could later drop in a single page web app to parse it and make something fancy.

**Comment by VikingPingvin** [comment]
Exactly, I was planning on creating a REST API anyways (as a way for the Agent to query and verify available files/configurations and access as well).
